capture.sh

```diff
@@ -11,9 +20,33 @@ else
   exit 1
 fi
 
-# screen captures
+PERIOD=4 # Default to 4 seconds between captures. Can be changed with --period command line option
+if [ ! -z "${period}" ]; then
+  if ! [ "$period" -eq "$period" ] 2>/dev/null || [ "$period" -eq "0" ]; then # Evaluate whether $period is a number and whether it is > 0
```

Comment:
I think this should be

```suggestion
  if ! [ "$period" -gt "0" ] 2>/dev/null; then # Evaluate whether $period is a number and whether it is > 0
```